app/code/Magento/Sales/i18n/en_US.csv

```diff
@@ -799,3 +799,5 @@ Refunds,Refunds
 "Allow Zero GrandTotal","Allow Zero GrandTotal"
 "Could not save the shipment tracking","Could not save the shipment tracking"
 "Please enter a coupon code!","Please enter a coupon code!"
+"Please enter a coupon code!","Please enter a coupon code!"
```

Comment:
@edenduong Could you remove the duplicate phrase?

Reply:
@engcom-Alfa: it is ok now. Thanks!